pkg/controller/certificates/certificate_controller_test.go

```diff
@@ -64,7 +64,8 @@ func TestCertificateController(t *testing.T) {
 
 	stopCh := make(chan struct{})
 	defer close(stopCh)
-	go informerFactory.Start(stopCh)
+	informerFactory.Start(stopCh)
+	informerFactory.WaitForCacheSync(stopCh)
```

Comment:
Actually I don't think we should return from this until Add Update or Delete is called on the DeltaFIFO, so pretty sure this waits for work in the queue. Since we only have one item of work. That should be sufficient.

https://github.com/kubernetes/kubernetes/blob/v1.6.4/staging/src/k8s.io/client-go/tools/cache/delta_fifo.go#L169